src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -135,7 +135,7 @@ class DatagridCell extends React.Component {
     const selected = selectedRowKey && context.rowKeyField(row) === selectedRowKey;
 
     return {
-      [type]: true,
+      ...(type && type !== 'undefined' ? { [type]: true } : {}),
```

Comment:
```suggestion
      ...(type ? { [type]: true } : {}),
```

Reply:
As you see here  `'undefined'` is a string not value `undefined`.  This will not work as expected because it will add class with type `DatagridCell--undefined`